### DISCUSSION TITLE
出现这个报错之后便闪退怎么办，求助佬

### DISCUSSION THREAD

**Discussion by Chtholly0721** [mikumifa/cppTickerBuy#12] [提问专区]
![880eb1bc3d8af0b4306e4fcde2cac348](https://github.com/user-attachments/assets/4f95d96b-3b50-480f-9f16-7b86b538e65e)

**Comment by mikumifa** [comment]
别开梯子

**Reply by Chtholly0721** [reply]
解决了！谢谢佬！
